templates/azuredisk-csi-driver.yaml

```
@@ -0,0 +1,24 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: azuredisk-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: azuredisk-csi-driver
+    kubeaddons.mesosphere.io/provides: csi-driver
+  annotations:
```

We have a temporarily required annotation for Addons in this repository to be compatible with Catalog:

```suggestion
  annotations:
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.4.0-1"
```